gateway/main/settings.py

```diff
@@ -367,12 +368,26 @@
     os.environ.get("RAY_CLUSTER_NO_DELETE_ON_COMPLETE", False)
 )
 
+RAY_CLUSTER_CPU_NODE_SELECTOR_LABEL = os.environ.get(
+    "RAY_CLUSTER_CPU_NODE_SELECTOR_LABEL",
+    "ibm-cloud.kubernetes.io/worker-pool-name: default",
+)
+
+RAY_CLUSTER_GPU_NODE_SELECTOR_LABEL = os.environ.get(
+    "RAY_CLUSTER_GPU_NODE_SELECTOR_LABEL",
+    "ibm-cloud.kubernetes.io/worker-pool-name: gpu-workers",
+)
+
 PROGRAM_TIMEOUT = int(os.environ.get("PROGRAM_TIMEOUT", "14"))
 
 GATEWAY_ALLOWLIST_CONFIG = str(
     os.environ.get("GATEWAY_ALLOWLIST_CONFIG", "api/v1/allowlist.json")
 )
 
+GATEWAY_GPU_JOBS_CONFIG = str(
```

Comment:
Similar to my previous comment, should we add this environment variable to the helm configuration?

Reply:
I'd wait until we're ready to make it a configmap... for now, easier to manage the file directly  in the internal repo

Reply:
Ok, you are planning to do the tests in the internal repo. Just to double check that I would not like to expose this info in the public repo.